records/hic_hal/lpc55s26.yaml

```diff
@@ -0,0 +1,74 @@
+common:
+    target:
+        - cortex-m33
+    core:
+        - Cortex-M33
+    macros:
+        - INTERFACE_LPC55XX
+        - CPU_LPC55S26JBD64
+        - DAPLINK_HIC_ID=0x4C504355  # DAPLINK_HIC_ID_LPC55XX
+        - OS_CLOCK=96000000
+    includes:
+        - source/hic_hal/nxp/lpc55xx
+        - source/hic_hal/nxp/lpc55xx/drivers
+        - source/hic_hal/nxp/lpc55xx/LPC55S26
+    sources:
+        hic_hal:
+            - source/hic_hal/nxp/lpc55xx
+            - source/hic_hal/nxp/lpc55xx/drivers
+            - source/hic_hal/nxp/lpc55xx/LPC55S26
+
+tool_specific:
+    uvision:
```

Comment:
`armcc` and `uvision` do not support Cortex-M33 so those section are not needed. You can also remove `source/hic_hal/nxp/lpc55xx/LPC55S26/armcc`.